src/OrbitGl/FunctionsDataView.cpp

```diff
@@ -58,8 +58,21 @@ bool FunctionsDataView::ShouldShowSelectedFunctionIcon(OrbitApp* app,
 }
 
 bool FunctionsDataView::ShouldShowFrameTrackIcon(OrbitApp* app, const FunctionInfo& function) {
-  return app->IsFrameTrackEnabled(function) ||
-         (app->HasCaptureData() && app->HasFrameTrackInCaptureData(function));
+  if (app->IsFrameTrackEnabled(function)) {
```

Comment:
@ronaldfw, @karupayun Is this still as intended? I remember the very recent changes here.

Reply:
made it a little bit more readable.

Reply:
Thanks